add the option to turn off ticks mirrored

phyjswang opened the pull request:
Dear Prof. Liu,

Thanks for this wonderful package!

For the APS theme, `theme.Axis.yticksmirrored` is always `true`. But when using left and right y axis for different quantities, one must turn off this option. Hence I add the key word argument `xticksmirrored` and `yticksmirrored` to make them optionally `false`.

Best,
Junsen Wang

phyjswang committed:
optionally turn off ticksmirrored

phyjswang committed:
optionally turn off ticksmirrored

liuyxpp commented:
@phyjswang will you have time to do the revisions? Thanks!

phyjswang reviewed (commented):
Ready to be merged.

liuyxpp commented:
> Ready to be merged.

Sorry. I am not aware of any new commit addressing the review I made above.

phyjswang commented:
> > Ready to be merged.
> 
> Sorry. I am not aware of any new commit addressing the review I made above.

Forgive me for not very familiar with GitHub PR stuffs. What is your review? I do not find it.

phyjswang committed:
change Makie to 0.21

phyjswang committed:
change MathTeXEngine to 0.6

liuyxpp reviewed (changes requested):
It would be better if these two options are explicitly listed as keyword args of theme_aps with default value true. Once you make this modification, I will merge the PR. Thanks!

phyjswang committed:
Update acs.jl

phyjswang committed:
Update aps.jl

make xticksmirrored and yticksmirrored as keyword args of theme_aps with default value true

phyjswang commented on src/aps.jl:
OK, I've made the changes.

phyjswang committed:
update compat

liuyxpp reviewed (commented):
I plan to merge this PR. Before that, can I ask why are you restrict Makie to v0.22 and Colors to v0.13? Shall we instead allow both Makie v0.21 and v0.22, Colors v0.12 and v0.13?

phyjswang commented:
> I plan to merge this PR. Before that, can I ask why are you restrict Makie to v0.22 and Colors to v0.13? Shall we instead allow both Makie v0.21 and v0.22, Colors v0.12 and v0.13?

There is no particular reason to restrict Makie and Colors to these version. You may freely change these version restrictions.
